nautobot/ui/src/nautobot.config.json

```diff
@@ -0,0 +1,3 @@
+{
+    "enabled-routes": ["/dcim/devices/", "/ipam/ip-addresses/"]
```

Comment:
Not 100% sure, if these are the only new-ui ready routes 🥸

Reply:
With #4016 and others, I would check the 2.0 milestone for which views we're going to polish before we ship.